### ISSUE TITLE
ReqAcct is all zero for HTTP/2

### ISSUE THREAD

**Issue by xcir** [varnishcache/varnish-cache#2238]
## Expected Behavior
varnishncsa %I and %O option output correctly in HTTP/2 request.

## Current Behavior
Request by HTTP/1.1.
```
# varnishncsa -F "%r I=%I O=%O"
GET http://***/munin.png HTTP/1.1 I=460 O=5160
GET http://***/munin.png HTTP/1.1 I=460 O=5156
GET http://***/munin.png HTTP/1.1 I=460 O=5160
```
Request by HTTP/2.0.
```
# varnishncsa -F "%r I=%I O=%O"
GET http://***/munin.png HTTP/2.0 I=0 O=0
GET http://***/munin.png HTTP/2.0 I=0 O=0
GET http://***/munin.png HTTP/2.0 I=0 O=0
```
Request by HTTP/2.0. (varnishlog)
```
# varnishlog -graw -ireqacct
        46 ReqAcct        c 0 0 0 0 0 0
     65551 ReqAcct        c 0 0 0 0 0 0
     65550 ReqAcct        c 0 0 0 0 0 0
        49 ReqAcct        c 0 0 0 0 0 0
```

## Possible Solution

Set the value to ReqAcct.

## Steps to Reproduce (for bugs)
See the Current Behavior above.

## Context
There is no way to know total bytes.

## Your Environment

* Version used: hitch1.4.4 + varnish-trunk revision bffb15e
* Operating System and version: Ubuntu 16.04.2 LTS (4.4.0-64-generic)

**Comment by bsdphk** [comment]
This is on purpose for now, because we need to decide how to count things in H2 relative to H1 to give meaningful statistics.

The "protocol overhead" of H1 is nearly non-existent, but in H2 it can be a significant fraction of the traffic and we must decide how we "charge" that in VSL and VSC.

**Comment by bsdphk** [comment]
This shold be fixed by 8abfe28c8925cf2541f757edce43c3bdd3952477
